web/src/upload.js

```diff
@@ -185,16 +177,36 @@ export async function upload_files(uppy, config, files) {
                 get_item("textarea", config),
             );
             compose_ui.autosize_textarea(get_item("textarea", config));
-            uppy.addFile({
+            file.id = uppy.addFile({
                 source: get_item("source", config),
                 name: file.name,
                 type: file.type,
                 data: file,
             });
         } catch {
             // Errors are handled by info-visible and upload-error event callbacks.
-            break;
+            continue;
         }
+
+        // TODO(before PR merges): what should we do when there's a lot of uploads at once?
```

Comment:
I asked on CZO about this [here](https://chat.zulip.org/#narrow/stream/101-design/topic/multiple.20uploads/near/1541526).